src/forscape_static_pass.cpp

```diff
@@ -1732,7 +1745,16 @@ ParseNode StaticPass::resolvePower(ParseNode pn){
             //return ast.setComplement(base);
 
         case OP_MAYBE_TRANSPOSE:
-            warnings.push_back(Error(parse_tree.getSelection(rhs), TRANSPOSE_T));
+            switch (Program::instance()->settings.warningLevel<WARN_TRANSPOSE_T>()) {
+                case WarningLevel::ERROR:
+                    return error(pn, rhs, TRANSPOSE_T);
+                case WarningLevel::WARN:
+                    warnings.push_back(Error(parse_tree.getSelection(rhs), TRANSPOSE_T));
+                    active_model->warnings.push_back(Error(parse_tree.getSelection(rhs), TRANSPOSE_T));
+                    break;
+                default: break;
```

Comment:
The error/warning system is such a mess. You might even consider fixing it on this MR.

Reply:
No, fix it in the next MR. It's scope creep